docker-compose.uffizzi.yml

```
@@ -0,0 +1,17 @@
+version: "3.0"
+
+# uffizzi integration
+x-uffizzi:
+  ingress:
+    service: memos
+    port: 5230
+
+services:
+  memos:
+    image: "${MEMOS_IMAGE}"
+    volumes:
+      - memos_volume:/var/opt/memos
+
```

We should set dev mode in PR preview to seed some demo data.
```suggestion
services:
  memos:
    image: "${MEMOS_IMAGE}"
    volumes:
      - memos_volume:/var/opt/memos
    command: ["--mode", "dev"]

```